datasets/integration_tests/pytorch/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "fds-integration-pytorch"
+version = "0.1.0"
+description = "Flower Datasets with PyTorch"
+authors = ["The Flower Authors <hello@flower.dev>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+flwr = "^1.5"
+torch = "^1.12.0"
+torchvision = "^0.14.1"
+flwr_datasets = { path = "./../../", develop = true, extras = ["vision"] }
```

Comment:
```suggestion
flwr-datasets = { path = "./../../", develop = true, extras = ["vision"] }
```

Reply:
Also, should we move this to right after `flwr`?

Reply:
Yes, I'll do that

Reply:
Actually, flwr shouldn't be listed there. I removed it.